beep/run_model.py

```diff
@@ -600,10 +599,11 @@ def main():
     try:
         args = docopt(__doc__)
         input_json = args['INPUT_JSON']
+        # TODO: resolve test file dir
```

Comment:
So one more note here, the pip build is failing because the default models are in the TEST_FILE_DIR.  Generally, we should never import the TEST_FILE_DIR in any non-test module because we want to be able to separate functional from testing pieces of data and code.  I've moved the models into their own directory and updated the manifest/setup.py accordingly.  Let me know if this is okay and I'll resolve the #TODO.

Reply:
Agreed, and thanks for cleaning that up. I am ok with the changes.